scripts/run.sh

```diff
@@ -177,14 +178,18 @@ mkdir -p ${RECO_TEMP}
 
 # Data egress to directory
 if [ "${COPYFULL:-false}" == "true" ] ; then
-  # Token for write authentication
-  export BEARER_TOKEN=$(cat ${_CONDOR_CREDS}/eic.use)
-  if [ -n ${XRDWURL} ] ; then
-    xrdfs ${XRDWURL} mkdir -p ${XRDWBASE}/${FULL_DIR} || echo "Cannot write simulation outputs to xrootd server"
+  if [ "${USERUCIO:-false}" == "true" ] ; then
+    python /opt/campaigns/hepmc3/scripts/register_to_rucio.py -f "${FULL_TEMP}/${TASKNAME}.edm4hep.root" -d "/${FULL_DIR}/${TASKNAME}.edm4hep.root" -s epic
```

Comment:
Does it make sense to hardcode the register script path like this? Can we use a `realpath $(dirname $0)` or so?

Reply:
hmmmm. trying to think. we could strip it from the RUCIO_CONFIG variable?